{{cookiecutter.project_slug}}/templates/404_multilingual.html

```diff
@@ -0,0 +1,28 @@
+{% load i18n %}
```

Comment:
Do we need a second template for this, or could the standard 404 be tweaked to include language code, direction, etc?

Don't forget the trans tag for all the text also!